cabal-install-solver/src/Distribution/Solver/Modular/Message.hs

```diff
@@ -1,8 +1,11 @@
 {-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE InstanceSigs #-}
 
 module Distribution.Solver.Modular.Message (
     Message(..),
-    showMessages
+    SummarizedMessage(..),
```

Comment:
I don't think that `SummarizedMessage` needs to be re-exported from this module.